### PR TITLE
CC 650 terraform profiles

### PR DESCRIPTION
**Pull request by mattch1**
Added placeholders for all except ci-dev - there do not seem to be any api url references in other webapps, so have not included:

NODE_PORT
EXTENSIONS_API_URL
EXTENSIONS_PROCESSOR_API_URL

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+api_local_url = "https://api.cidev.aws.chdev.org"
+authentication_middleware = "on"
+cache_server = "cidev-chs-elasticache.9vgugg.ng.0001.euw2.cache.amazonaws.com:6379"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chs_api_key = "OpXFFV77ghkgZlK9U7HcVvaK0rUqKUK0M0AlDi5c"
```
Comment (mattch1) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:14]: Not sure whether this is the correct value or not.

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+api_local_url = "https://api.cidev.aws.chdev.org"
+authentication_middleware = "on"
+cache_server = "cidev-chs-elasticache.9vgugg.ng.0001.euw2.cache.amazonaws.com:6379"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
```
Comment (mattch1) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:13]: And this one.

Comment (markpit) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:13]: the cdn_host seems to match the value we have in cidev profile for overseas-entities-web so should be ok

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (markpit) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:14]: i have a feeling that might need to be moved to vault/secret

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+chs_api_key = "OpXFFV77ghkgZlK9U7HcVvaK0rUqKUK0M0AlDi5c"
+chs_url = "https://cidev.aws.chdev.org"
+cookie_domain = "cidev.aws.chdev.org"
+cookie_expiration_in_seconds = "3600"
+cookie_name = "__SID"
+cookie_secret = "ChGovUk-XQrbf3sLj2abFxIY2TlapsJ"
```
[the remark this answers is not in the document]
Comment (markpit) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:19]: i'm guessing as it is a secret it should go in vault, add @tobafolaranmi-ch he might be able to tell us

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:14]: Cannot see it in the profiles or in the vault section of overseas_entities_web, only in locals.

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 3 above)
[the remark this answers is not in the document]
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Removed:
authentication_middleware
cache_server
chs_api_key
cookie_secret

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+kms_alias = "alias/development-eu-west-2/environment-services-kms"
+
+# service configs
+api_local_url = "https://api.cidev.aws.chdev.org"
+authentication_middleware = "on"
+cache_server = "cidev-chs-elasticache.9vgugg.ng.0001.euw2.cache.amazonaws.com:6379"
```
[the remark this answers is not in the document]
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Removed this field.

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (mattch1) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: Removed this field.

**Review by markpit — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**